policies/postgres/policies.go

```diff
@@ -123,6 +123,37 @@ func (r policiesRepository) SaveDataset(ctx context.Context, dataset policies.Da
 
 }
 
+func (r policiesRepository) UpdateDatasetToInactivate(ctx context.Context, groupID string, ownerID string) error {
```

Comment:
let's call this `InactivateDatasetByGroupID` since we will need other ways to inactivate in the future